small_instruments/libraries/centrifuges/source.rb

```diff
@@ -14,7 +14,7 @@ module Centrifuges
   # @param type [String] the type of pipettor if a specific one is desired
   # @return [String] directions
   def spin_down(items:, speed:, time: nil, type: nil)
-    is_plate = items.any?(&:collection?)
+    is_plate = items.any? { |item| item.respond_to?('collection') }
```

Comment:
Not clear what you're trying to find out here. I would think that they **all** have to be a collection or not a collection to spin them down at the same time.

Reply:
Also, I would try to use the `collection?` or `is_part` method to get this information, if possible.